packages/@okta/vuepress-site/code/react-native/index.md

```diff
@@ -28,7 +28,7 @@ These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widge
 
 * [okta-react-native on npm](https://www.npmjs.com/package/@okta/okta-react-native)
 * [React Native SDK Source](https://github.com/okta/okta-react-native)
-* [React Native redirect authentication sample app](https://github.com/okta/samples-js-react-native): See [Browser Sign In](https://github.com/okta/samples-js-react-native/tree/master/browser-sign-in) for a redirect configuration.
+* [React Native redirect authentication sample app](https://github.com/okta/samples-js-react-native): See [Browser sign-in](https://github.com/okta/samples-js-react-native/tree/master/browser-sign-in) for a redirect configuration.
```

Comment:
Actually, neither are right as we don't use sign-in as a noun. It would need to be Browser sign-in flow

Reply:
Changed to browser sign-in flow